data/test/scenarios/recall_by_unit_tag.cfg

```diff
@@ -1,13 +1,19 @@
 # wmllint: no translatables
 
-# The [unit] tag can recall units if a unit with that id already exists
-# on the recall list. Some edge cases were unclear, so this test finds
-# out what they do.
-#
 # This is a test where any failure should probably be fixed by changing the
 # test to match changes in the engine. Having the test will be useful for
 # reference to find out when the engine behavior changed.
```

Comment:
This is another where the comment is becoming detached from the test that it's documentation for.

Reply:
Resolved as a non-issue because this is another where it's the only test in that file.